documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+
+   ```
+   https://${HOSTNAME}:${SSLPORT}/em
+   ```
+
+#### Use an Administration Server `NodePort`
+
+1. Configure the Administration Server to expose an externally accessible NodePort in the Domain resource.
+
+   For an example of setting up the `NodePort` on an Administration Server, see [Use a `NodePort`]({{< relref "/managing-domains/accessing-the-domain/wlst#use-a-nodeport" >}}). For information about the `NodePort` Service on an Administration Server, see the [Domain resource](https://github.com/oracle/weblogic-kubernetes-operator/blob/{{< latestMinorVersion >}}/documentation/domains/Domain.md) document.
+
+2. Use the following URL from your browser to access the WebLogic Server Administration Console:
+
+   ```
+   http://hostname:adminserver-NodePort/console
+   ```
+   The `adminserver-NodePort` is the port number of the Administration Server outside the Kubernetes cluster.
+
+   If you have an [FMW Infrastructure]({{< relref "/managing-fmw-domains.md" >}}) type domain, then you can also access the Fusion Middleware Control (Enterprise Manager) Console using the following URL:
+   
+   ```
+   http://hostname:adminserver-NodePort/em
+   ```
+
+#### Use a `kubectl port-forward` connection
+
+1. Forward a local port (that is external to
+   Kubernetes) to the administration port of the
+   Administration Server Pod according to these
+   [instructions]({{< relref "/managing-domains/accessing-the-domain/port-forward.md" >}}).
+
+   **NOTE:** If you plan to access the WebLogic Server Administration Console from a browser
+   on a different machine than the port forwarding command,
+   then the port forwarding command needs to specify a `--address` parameter
+   with the IP address of the machine that is hosting the command.
+
+1. Use the below URL using either the local hostname or IP address
+   from the `port-forward` command in the first step, plus the local port from
+   this same command to access the WebLogic Server Administration Console. For example:
+
+   ```
+   http://${HOSTNAME}:${LOCAL_PORT}/console
+   ```
+   Where:
+
+     * `${HOSTNAME}` is the hostname or the defined IP address of the machine
+       where the `kubectl port-forward` command is running. This is
+       customizable on the `port-forward` command and is `localhost`
+       or `127.0.0.1`, by default.
+
+     * `${LOCAL_PORT}` is the local port where the `kubectl port-forward` command is running.
+       This is specified on the `port-forward` command.
+
+   If you have an [FMW Infrastructure]({{< relref "/managing-fmw-domains.md" >}}) type domain, then you can also access the Fusion Middleware Control (Enterprise Manager) Console using the following URL:
+
+   ```
+   http://${HOSTNAME}:${LOCAL_PORT}/em
+   ```
+
+### Test 
+To verify that your load balancer, `NodePort`, or `kubectl port-forward` setup is working as expected, 
+see [Test]({{< relref "/managing-domains/accessing-the-domain/admin-console#test" >}}).
```

Comment:
This Test seems specific to the remote console. Is it certain that'll work for the admin console too?

Reply:
I believe it'll work with console as well if you using the path-based routing. The 'curl' command is already specifying username and password and 'host' is not needed for path-routing.